src/styles/components/_navbar.scss

```diff
@@ -236,7 +236,7 @@ html[data-theme='dark'] {
 
             border: none;
 
-            background: var(--ifm-menu-link-sublist-icon) 50% / 0.5rem 0.25rem;
+            background: var(--ifm-menu-link-sublist-icon) 50% / 0.5rem 0.3rem;
```

Comment:
Docusaurus v3 causes the version dropdown icon to be smaller. This change makes sure that it looks like the current docs site.


| main | Docusaurus v3 (w/o the change) |
|--------|--------|
|  ![Screenshot 2024-01-03 at 1 23 16 PM](https://github.com/ionic-team/ionic-docs/assets/13530427/09b7eb19-d9a0-490d-a62b-9c1ebd4ad20b) |  ![Screenshot 2024-01-03 at 1 23 45 PM](https://github.com/ionic-team/ionic-docs/assets/13530427/1f29c09b-3119-4da0-9404-168bac1ec4b1) |